modules/gltf/docs/api-reference/gltf-scenegraph.md

```diff
@@ -38,6 +40,48 @@ const scenegraph = gltf.getScene();
 const scenegraph = gltf.getScene(2);
 ```
 
+### Modifying
+
+```js
+import {load, encode} from '@loaders.gl/core';
+import {ImageWriter} from '@loaders.gl/images';
+import {GLTFLoader, GLTFWriter, GLTFScenegraph} from '@loaders.gl/gltf';
+
+// Load and parse a file
+const gltfData = await load(GLTF_URL, GLTFLoader);
+
+const meshIndex = gltfBuilder.addMesh(gltfData.meshes[0].primitives[0].attributes);
+const nodeIndex = gltfBuilder.addNode(meshIndex);
+const sceneIndex = gltfBuilder.addScene([nodeIndex]);
+gltfBuilder.setDefaultScene(sceneIndex);
+const imageBuffer = await encode(gltfData.images[0].image, ImageWriter);
+const imageIndex = gltfBuilder.addImage(imageBuffer, 'image/jpeg');
+const textureIndex = gltfBuilder.addTexture(imageIndex);
+const pbrMaterialInfo = {
+  pbrMetallicRoughness: {
+    baseColorTexture: textureIndex
+  }
+};
+gltfBuilder.addMaterial(pbrMaterialInfo);
+gltfBuilder.createBinaryChunk();
+
+const gltfBuffer = encodeSync(gltfBuilder.gltf, GLTFWriter);
+```
+
+## Fields
+
+### gltf: Object
+
+Contains all data of gltf including the json part and the binary chunk. The binary chunk is generated by `createBinChunk()` method
+
+### sourceBuffers: Array<ArrayBuffer | { arrayBuffer?: ArrayBuffer; buffer?: ArrayBuffer; byteOffset: number; byteLength: number}>
```

Comment:
I think you will need to put backticks around this otherwise our website builder will interpret the `<` character as MDX syntax.

```md
### `sourceBuffers: Array<ArrayBuffer | { arrayBuffer?: ArrayBuffer; buffer?: ArrayBuffer; byteOffset: number; byteLength: number}>`
```